src/request/create/Create.css

```diff
@@ -1,63 +1,77 @@
-.page .actionbox p {
-    max-width: 32rem;
-    margin-left: auto;
-    margin-right: auto;
-}
-
+.page#set-passphrase .page-body,
 .page#choose-identicon .page-body {
     position: relative;
+    display:flex;
+    flex-direction: column;
+    align-items: center;
 }
 
-.page#choose-identicon .page-body p:last-of-type {
-    margin-bottom: 2rem;
+.page#set-passphrase .page-body h2,
+.page#choose-identicon .page-body h2 {
+    text-align: center;
+    font-weight: normal;
 }
 
-.page-footer {
-    padding: 1rem;
+.page#set-passphrase.repeat-password .hide-on-repetition {
+    display: none;
 }
 
-.keyfile-info {
-    display: flex;
-    flex-direction: row;
+.page#set-passphrase:not(.repeat-password) .show-on-repetition {
+    display: none;
 }
 
-.keyfile-info p:first-child {
-    margin-top: 0;
+.page-footer {
+    padding: 1rem;
 }
 
-.keyfile-info p:last-child {
-    margin-bottom: 0;
+.page-header {
+    padding-top: 5.5rem;
+}
+
+.page-header .progress-indicator {
+    position: absolute;
+    left: 0;
+    top: 0;
 }
 
-.nq-icon.walletfile {
-    width: 9rem;
-    height: 11.75rem;
-    margin-right: 4rem;
-    flex-shrink: 0;
+.page-header .page-header-back-button {
+    top: 5.5rem;
 }
 
-.page#recovery-words .skip-container {
+.wallet-identifier {
+    width: 20rem;
+    height: 20rem;
     text-align: center;
 }
 
-.page#recovery-words .skip {
-    color: inherit;
-    cursor: pointer;
+.wallet-identifier .identicon {
+    width: 7rem;
+    height: 6rem;
+    position: absolute;
+    transform: translate(0.2rem, 3.5rem) scale(0);
+    animation: pop-in-13 0.4s cubic-bezier(.73,2.08,.48,1);
```

Comment:
What does the `-13` in the animation name stand for?

Reply:
renamed to `pop-in-0-to-115`